src/javascript/app/pages/cashier/payment_agent_withdraw.js

```diff
@@ -197,8 +190,19 @@ const PaymentAgentWithdraw = (() => {
                 $('#successMessage').css('display', '')
                     .attr('class', 'success-msg')
                     .html($('<ul/>', { class: 'checked' }).append($('<li/>', { text: localize('Your request to withdraw [_1] [_2] from your account [_3] to Payment Agent [_4] account has been successfully processed.', [request.currency, getNumberFormat(request.amount, request.currency), Client.get('loginid'), agent_name]) })));
+                
+                // Set PA details.
+                document.querySelector('#agentName').innerHTML = agent_name;
```

Comment:
why not getelementbyid? queryselector is the same as using jquery. we have a function to getelementbyid that returns an empty element if it doesn't exist so you don't need to check it every time.

Reply:
@negarn What do you mean by "check it every time"?